ci-operator/step-registry/ipi/conf/additional-ca/trust-policy/ipi-conf-additional-ca-trust-policy-ref.yaml

```
@@ -0,0 +1,21 @@
+ref:
+  as: ipi-conf-additional-ca-trust-policy
+  from_image:
+    namespace: ocp
+    name: "4.12"
+    tag: upi-installer
+  commands: ipi-conf-additional-ca-trust-policy-commands.sh
+  resources:
+    requests:
+      cpu: 10m
+      memory: 100Mi
+  credentials:
+  - namespace: test-credentials
+    name: openshift-custom-mirror-registry
+    mount_path: /var/run/vault/mirror-registry
+  env:
+  - name: ADDITIONAL_CA_POLICY
```

```suggestion
  - name: ADDITIONAL_TRUST_BUNDLE_POLICY
```
To keep consistent with key name in install-config